experimental/AlgebraicStatistics/src/MarkovProperties.jl

```diff
@@ -184,6 +184,19 @@ function is_acyclic(G::Graph{Directed})::Bool
   return true
 end
 
+function topological_sort(G::Graph{Directed})::Vector{Int}
```

Comment:
we should move this into src/Combinatorics/Graphs/functions.jl in a separate PR. is_acyclic uses a very similar algorithm so we can avoid some code duplication here.

Reply:
Do you mean we should put all the general graph theory functions into `Combinatorics` while our branch is merged into Oscar master? Or should these functions go into Oscar earlier already?

Reply:
Ideally this should be a PR before we merge to Oscar master.

In the end we should have seperate PRs for any changes going into the src folder, and ultimately means we need to merge them first.

Reply:
but, ok, this is not exactly something that is already in src and I think can be resolved afterwards as well.
So maybe just leave a TODO comment